### PR TITLE
Update mush.

### PR DESCRIPTION
**Pull request by mariaum**
## Type of change

- [ ] Add new directory for a server
- [X] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/mush/manifest.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "direct_ip": "jogar.mush.com.br",
   "server_wildcards": [
     "%.mush.com.br",
```
Comment (DoJapHD) [inline minecraft_servers/mush/manifest.json:6]: Is not a real wildcard
https://i.imgur.com/oUoCLHY.png

File: minecraft_servers/mush/manifest.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
...
-    "%.mushmc.com"
+    "%.mushmc.com",
+    "%.muu.sh"
```
Comment (DoJapHD) [inline minecraft_servers/mush/manifest.json:8]: Not a real wildcard
https://i.imgur.com/SL6r8Yc.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/mush/manifest.json (hunk 1 above)
Comment (mariaum) [reply minecraft_servers/mush/manifest.json:6]: Wildcard should be working now.

File: minecraft_servers/mush/manifest.json (hunk 2 above)
Comment (mariaum) [reply minecraft_servers/mush/manifest.json:8]: Wildcard should be working now.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**